joomla/joomla-cms
# [4.0] Toggle Password Visibility

Now that the code has been fixed with #27412 I can update the strings to be more meaningful.

The strings for Screen Readers are now "Show Password" and "Hide Password"

### Testing Instructions
Make sure you are the current codebase (ie you have #27412)

Apply this PR

Update the javascript with
npm i 

Inspect the password field in the admin user manager, admin login, site login

The strings for Screen Readers are now "Show Password" and "Hide Password"
